### ISSUE TITLE
How can you detect a browser window close event?

### ISSUE THREAD

**Issue by ssmereka** [electron/spectron#257]
In our application a loading window is presented, then closed and a new window shown.  I'm trying to add a listener for the window close event, but I cannot get it working.  How would I do this with Spectron?

e.g.  In electron you can do the following:
```javascript
let win = new BrowserWindow({width: 800, height: 600})
win.on('closed', () => {
  console.log('window closed')
})
```

but, in Spectron the following doesn't work:
```javascript
this.app.client.browserWindow.on('close', () => {
  console.log('window closed')
})
```

**Comment by meekaah** [comment]
yeah im stuck on this too... if anyone has any idea that'd be great

**Comment by mahchiahui** [comment]
I also have the same problem. Have you guys found a fix?

**Comment by CipherTooth** [comment]
I'm using the electron windowManager, had the same problem, then I assigned my call back to the object property... Hope it helps... (someone)

**Comment by moeinghasemi381** [comment]
I have the same problem!
